css/desktop.css

```diff
@@ -0,0 +1,309 @@
+  vertical-align: middle;
+}
+
+.headline {
+  height: 870px;
+  background-image: url("../image/desktop.png");
+  background-repeat: no-repeat;
+  background-size: fill;
+  background-position: right 1px top 1px;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  column-gap: 87px;
+  padding: 0 36px;
+}
+
+.headline-top-section {
+  display: flex;
+  flex-direction: column;
+  width: 560px;
+  padding: 192px 0;
+}
+
+.social-media {
+  order: -1;
+  align-self: flex-start;
+  display: flex;
+  flex-direction: column;
+  padding: 360px 0;
+  justify-content: center;
+}
+
+.title {
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 81.8px;
+}
+
+.supporting {
+  font-size: 32px;
+  font-style: medium;
+  letter-spacing: 0.03em;
+  line-height: 44px;
+}
+
+.description {
+  font-size: 20px;
+  line-height: 32px;
+  font-style: regular;
+  letter-spacing: 0.03em;
+  font-weight: 400;
+}
+
+.work {
+  margin: 0 118px;
+  padding: 0 24px;
+}
+
+.work-header {
+  display: flex;
+  justify-self: start;
+  justify-content: center;
+  align-items: baseline;
+  column-gap: 24px;
+  margin-top: 135px;
+  margin-bottom: 80px;
+}
+
+.work-header .title {
+  font-size: 40px;
+  margin: 10px 10px;
+}
+
+.work-header hr {
+  width: 794px;
+  margin: 8px 0;
+  height: 1px;
+  padding-right: 100px;
+}
+
+.feature-section {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 24px;
+}
+
+.feature-image {
+  width: 50%;
+}
+
+.feature-image img {
+  width: 100%;
+}
+
+.story-section {
+  grid-column: 8/13;
+  width: 448px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 24px;
+}
+
+.story-title {
+  font-size: 45px;
+  line-height: 52px;
+}
+
+.story-description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #091e42;
+}
+
+.card-section {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 24px;
+}
+
+.card {
+  width: 369px;
+  height: 435px;
+  padding: 8px 16px;
+}
+
+.card-body {
+  margin-top: 170px;
+}
+
+.card2 .card-button,
+.card3 .card-button,
+.card4 .card-button,
+.card5 .card-button,
+.card6 .card-button {
+  display: none;
+}
+
+.card1 .card-body {
+  margin-top: 76px;
+  margin-bottom: 24px;
+}
+
+.card1:hover {
+  background-image: url("../image/card1.png");
+  background-repeat: no-repeat;
+  padding: 0;
+  display: grid;
+}
+
+.card2 .card-body {
+  margin-top: 170px;
+}
+
+.card1:hover .card-body {
+  display: none;
+}
+
+.card1 .card-button {
+  align-self: flex-end;
+  box-sizing: border-box;
+  height: 58px;
+  width: 100%;
+  text-decoration: none;
+  color: #fff;
+  padding: 10px;
+  text-align: center;
+}
+
+.card1:hover .card-button {
+  width: 100%;
+  height: 58px;
+  text-align: center;
+}
+
+.card1 .card-button:hover {
+  text-decoration: none;
+  color: #f55800;
+}
+
+.card2,
+.card5 {
+  background-image: url("../image/card2.png");
+  background-repeat: no-repeat;
+}
+
+.card3,
+.card6 {
+  background-image: url("../image/card3.png");
+  background-repeat: no-repeat;
+}
+
```

Comment:
- [ ] I am really glad for your effort in improving your project based on the previous reviewer's suggestions. You have done a great job so far 👏. In addition to what you have implemented, I would suggest **you implement what you have done on the project card `card1`, you implement them on the remaining 5 project cards.** That means from `card2` to `card6`. because all the project cards should have that hover effect as the previous reviewer suggested. so please, add that hover effect on card1 to the remaining 5 cards. 😃